[FEAT-12-13] Fix type CommnadsScheduler, Spefic Commands #12  #13 (#15)
taecontrol/larastats-docs: docs/uptime-check/uptime-check-commands.md

## CheckUptimeCommand

To add the CheckUptimeCommand to your Laravel application, follow these steps:

> Open the ``app/Console/Kernel.php`` file in your Laravel project.
> Now we import the CheckUptimeCommand class:: ``use Taecontrol\MoonGuard\Console\Commands\CheckUptimeCommand;``

Reviewer: ```suggestion
Open `app/Console/Kernel.php` file in your Laravel project and import the `CheckUptimeCommand` class: 

`use Taecontrol\MoonGuard\Console\Commands\CheckUptimeCommand;`
```
